### DISCUSSION TITLE
Is it possible to export new clips from Marker points

### DISCUSSION THREAD

**Discussion by asodivnaoiioiojasiodfj** [stashapp/stash#3004] [General]
I want to be able to export new clips from certain marker points. Is this possible?

**Comment by stg-annon** [comment]
it is possible in the sense that it can be done however it is not a currently supported feature of stash

This could be accomplished via a plugin for a known marker and duration you wish to stop after as stash markers don't currently support stop times

**Reply by asodivnaoiioiojasiodfj** [reply]
Thank you so much for the reply! What plugin could accomplish this?

**Reply by stg-annon** [reply]
none that I know of ATM but it would likely just need to pipe the start time from the marker from stash with a given user length into ffpmeg which would generate the clips, I personally am not too familiar with ffmpeg but I know that it can be done, stash kind of does this with the generate task but with fixed length previews
